src/types.ts

```diff
@@ -699,6 +728,11 @@ export type UserResponse<UserType = UnknownType> = User<UserType> & {
  * Option Types
  */
 
+export type MessageFlagsPaginationOptions = {
```

Comment:
looks like a base, don't we have it around ?

Reply:
not that I'm aware of